conda_smithy/configure_feedstock.py

```diff
@@ -102,10 +106,10 @@ def render_circle(jinja_env, forge_config, forge_dir):
 
 @contextmanager
 def fudge_subdir(subdir):
-    orig = conda_build.metadata.cc.subdir
-    conda_build.metadata.cc.subdir = subdir
+    orig = cc.subdir
+    cc.subdir = subdir
     yield
-    conda_build.metadata.cc.subdir = orig
+    cc.subdir = orig
 
 
```

Comment:
I don't think we have any other options right now.  Without a way of directly specifying this to conda, this is the best we can do.